### PR TITLE
December 2014

### PR DESCRIPTION
**Pull request by dantleech**
Please add your news here ..

### CODE REVIEW

File: _posts/2015-01-03-Update-December-2014.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Welcome to the 6th monthly PHPCR news update and the last of 2014.
+
+### CMF Meetup
+
+On the 12th of Decembed the CMF team metup in Würzburg, Germany and worked on
+Jackalope in amongst other things. We also discussed the next iteration of
+Jackalope, "Jackalope 2" - which would aim to be more modular and more
+performant.
+
+We hope to hold another meetup early this year.
```
Comment (dbu) [inline _posts/2015-01-03-Update-December-2014.md]: you could link the cmf mailinglist and tell people to sign up there if interested.

**Comment by dbu** [comment]
the jackalope 1.2 release should come soon. we should put in a last effort to wrap up things or throw them out of the milestones and release alpha. 

lets say: jackalope 1.2 release was delayed but will happen soon - remaining tasks 
https://github.com/jackalope/jackalope/milestones/1.2, https://github.com/jackalope/jackalope-doctrine-dbal/milestones/1.2 and https://github.com/jackalope/jackalope-jackrabbit/milestones/1.2

**Comment by dbu** [comment]
@dantleech want to publish this? or directly make it the january news?

**Comment by dantleech** [comment]
Lets make it January. Not sure if you received my mail on the list, but I think I suggested dumping this blog and instead do a monthly on the CMF website. CMF is a more logical place as it really covers PHPCR, Jackalope and the Symfony CMF.

**Comment by dantleech** [comment]
Also we can register a SymfonyCMF Twitter account :P

**Comment by dbu** [comment]
agreed. so lets make this the last post and put a note on the news page to go over to the cmf news. that would probably mean blogging on the symfony blog, right?

i just created https://twitter.com/SymfonyCMF - no idea how but would like to share that account with @lsmith77 and you.

**Comment by lsmith77** [comment]
shall we close this one then?

**Comment by dantleech** [comment]
Yeah. I will make a post saying that we will announce news on the CMF website and change the default homepage.

**Pull request closed without merging**